server/utils/secrets/secret_ref_utils.py

```
@@ -191,20 +191,22 @@ def has_user_credentials(self, user_id: str, provider: str) -> bool:
             cursor.execute(
-                """SELECT 1 FROM user_tokens
-                   WHERE (user_id = %s OR org_id = %s)
+                f"""SELECT 1 FROM user_tokens
+                   WHERE user_id = %s
                      AND provider = %s
                      AND secret_ref IS NOT NULL
                      AND is_active = TRUE
+                     {clause}
                    LIMIT 1""",
-                (user_id, org_id, provider)
+                (user_id, provider_base) + params,
```

Ruff (0.15.12)

[warning] Consider `(user_id, provider_base, *params)` instead of concatenation

Replace with `(user_id, provider_base, *params)`

(RUF005)